test/e2e/apps/disruption.go

```diff
@@ -215,6 +197,34 @@ var _ = SIGDescribe("DisruptionController", func() {
 			}
 		})
 	}
+
+	It("Allows updates to pdb's", func() {
```

Comment:
nit: should we instead use PDB's ?

Reply:
If you want to get super nit-picky, the plural of an initialism shouldn't have an apostrophe, unless the initialism is written with a period after each letter. So in this case it should be "PDBs".

Reply:
How about
`should block and allow evicting pod with PodDisruptionBudget`
like other tests in this test module?